setup.py

```diff
@@ -355,6 +355,9 @@ def cython_extensions_and_necessity():
     "futures>=2.2.0; python_version<'3.2'",
     "enum34>=1.0.4; python_version<'3.4'",
 )
+EXTRAS_REQUIRES = {
+    "protobuf": "protobuf>=3.5.0.post1",
```

Comment:
Any reason for this particular version?

Reply:
I copied it from our `requirements.txt`.